src/Form/Input/Input.stories.tsx

```diff
@@ -16,9 +16,20 @@ const StyledStory = styled('div')`
     padding: 2rem 5rem;
 `;
 
+type InputState = {
+    text?: string;
+    number?: string | number;
+    tel?: string | number;
+};
+
 const InputStories = () => {
     const { useState } = React;
-    const [someInputVal, setSomeInputVal] = useState<string>('');
+    const [inputValues, setInputValues] = useState<InputState>({
```

Comment:
This is probably better, but was I was actually asking about is why theres a need to have different state for each of them, as opposed to all just using the same state like before.

Reply:
I did that at first. If they all pull from a single state value then all the form fields update at the same time with the same value. Originally only the `text` input was using state, and typing into the other fields was giving an uncontrolled form field error.

Reply:
Oh derp. This is the story, not the component file. My bad, nothing to see here.